arlo-client/src/components/Audit/useSetupMenuItems/getJurisdictionFileStatus.ts

```diff
@@ -0,0 +1,40 @@
+import { api, checkAndToast } from '../../utilities'
+import { IErrorResponse } from '../../../types'
+
+export enum FileProcessingStatus {
+  ReadyToProcess = 'READY_TO_PROCESS',
+  Processing = 'PROCESSING',
+  Processed = 'PROCESSED',
+  Errored = 'ERRORED',
+}
+
+export interface IJurisdictionsFileResponse {
+  file: {
+    contents: null | string
+    name: string
+    uploadedAt: string
+  } | null
+  processing: {
+    status: FileProcessingStatus
+    startedAt: string | null
+    completedAt: string | null
+    error: string | null
+  } | null
+}
+
+const getJurisdictionFileStatus = async (
+  electionId: string
+): Promise<FileProcessingStatus> => {
+  const jurisdictionsOrError:
+    | IJurisdictionsFileResponse
+    | IErrorResponse = await api(`/election/${electionId}/jurisdiction/file`)
+  if (checkAndToast(jurisdictionsOrError)) {
+    return FileProcessingStatus.Errored
+  }
```

Comment:
Not necessary, per se, but my assumption is that it would be more confusing later on when trying to adjust the error handling to be consistent if it's missing. All api interactions on the front end have this piece, so we might as well leave it as a point from which to develop.